runner/cfd/cmd/drive/read_env.go

```diff
@@ -18,22 +18,22 @@ type EnvCfg struct {
 
 	ContainerId string `env:"CONTAINER_ID"`
 
-	RegUser string `env:"CUSTOM_ENV_CI_REGISTRY_USER"`
-	RegPass string `env:"CUSTOM_ENV_CI_REGISTRY_PASSWORD"`
-	JobImg  string `env:"CUSTOM_ENV_CI_JOB_IMAGE"`
+	JobImg         string `env:"CUSTOM_ENV_CI_JOB_IMAGE"`
+	DockerPass     string `env:"CF_DOCKER_PASSWORD"`
```

Comment:
[question] Do we need this? We set `CF_DOCKER_PASSWORD` with the value of either `DockerHubToken` or `CIRegistryPass`, we don't read it in.